apps/s_time.c

```diff
@@ -240,16 +224,10 @@ int s_time_main(int argc, char **argv)
                                    www_path);
             if (buf_len <= 0 || SSL_write(scon, buf, buf_len) <= 0)
                 goto end;
-            while ((i = SSL_read(scon, buf, sizeof(buf))) > 0 ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_READ ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_WRITE)
```

Comment:
so my conclusion is that reused sessions in tls1.3
cause the same cpu cycles for the client than new tls1.3 sessions,
just the server has significantly less cpu cycles.
OTOH tls1.2 session reuse may cause more round trips,
but much less cpu load on both server and client, and if
the network latency is very low as in this case the numbers
are not as convincing as they should be.
Is this by design, or could the TLS session have an
estimation of the network delay and choose another
strategy, if the round trips are cheap?